src/ape/api/networks.py

```diff
@@ -357,6 +362,18 @@ def get_network_data(self, network_name: str) -> Dict:
 
         return data
 
+    def get_proxy_info(self, address: AddressType) -> Optional[ProxyInfoAPI]:
+        """
+        Information about a proxy contract such as proxy type and implementation address.
+
+        Args:
+            address (str): The address of the contract.
+
+        Returns:
+            ProxyInfoAPI or None if the contract does not use any known proxy pattern.
```

Comment:
```suggestion
            Optional[:class:`~ape.api.networks.ProxyInfoAPI`]: Returns ``None`` if the contract 
            does not use any known proxy pattern.
```

This would link to the class so the user can see that it has an address field.
Only works